core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -962,6 +966,11 @@ object Parser {
     *  see @backtrack if you want to do backtracking.
     *
     *  This is the same as parsers.foldLeft(fail)(_.orElse(_))
+    *
+    *  recommended style: oneOf(p1 :: p2 :: p3 :: Nil) rather than
```

Comment:
Why do you care how people define their Lists?

Reply:
Using List(a, b, c) uses a var args constructor which allocates and then builds a list after. Using this style avoids that.

IMO it looks nicer to not have the List(..) present as well.

That is why I recommend it.